pkg/ttnpb/rights.go

```diff
@@ -73,6 +73,15 @@ func (r Right) Implied() *Rights {
 	return RightsFrom(r)
 }
 
+// PrettyName returns a user friendly form of the right
+func (r Right) PrettyName() string {
```

Comment:
We should use the `i18n` package instead of `PrettyName`. All rights already have friendly, translatable descriptions.